tests/acceptance/commons/glance_operations.py

```diff
@@ -134,29 +134,41 @@ def create_image(self, image_glance_name, image_filename=None, container_format=
             image.update(properties=custom_properties)
         return image.id
 
-    def update_image_properties(self, image_id, custom_properties):
+    def update_image_properties(self, image_id, **kwargs):
         """
         Update the image properties of the given image_id
         :param image_id: Image ID
```

Comment:
The documented parameters should be among the given _**kwargs_ parameters. If not, they are not considered.